docs/tutorials/data_manipulation/dataframesdataset.md

```diff
@@ -0,0 +1,375 @@
+# Pandas dataframe based dataset
```

Comment:
How about `Pandas datasets` only?

Reply:
I think that's too imprecise, but maybe we can do "`pandas.DataFrame` based dataset".